sdk_container/src/third_party/coreos-overlay/x11-drivers/nvidia-drivers/files/bin/setup-nvidia

```diff
@@ -24,15 +24,12 @@ function download_flatcar_developer_container() {
   then
     rm -f "${FLATCAR_ROOT_WORKDIR}/"flatcar_developer_container*.bin
     FLATCAR_REMOTE_FILENAME=${FLATCAR_DEVELOPER_CONTAINER%%-*}.bin.bz2
-    if [[ "$GROUP" == "developer" ]]
+    if [[ "$(curl -I -s -o /dev/null -w "%{http_code}" "https://${GROUP}.release.flatcar-linux.net/${FLATCAR_RELEASE_BOARD}/${FLATCAR_RELEASE_VERSION}/${FLATCAR_REMOTE_FILENAME}")" == 200 ]]
```

Comment:
```suggestion
    if [[ "$(curl -I -L -s -o /dev/null -w "%{http_code}" "https://${GROUP}.release.flatcar-linux.net/${FLATCAR_RELEASE_BOARD}/${FLATCAR_RELEASE_VERSION}/${FLATCAR_REMOTE_FILENAME}")" == 200 ]]
```
For future proofing we should also accept forwards

Reply:
Should we add `--retry-delay 1 --retry 60 --retry-connrefused --retry-max-time 60 --connect-timeout 20` to make it more robust against slow network setup during boot or other temporary network issues?